extensions/keycloak-admin-client-common/runtime/src/main/java/io/quarkus/keycloak/admin/client/common/KeycloakAdminClientConfig.java

```diff
@@ -0,0 +1,72 @@
+package io.quarkus.keycloak.admin.client.common;
+
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+/**
+ * Keycloak Admin Client
+ */
+@ConfigRoot(phase = ConfigPhase.RUN_TIME, name = "keycloak.admin-client")
+public class KeycloakAdminClientConfig {
+
+    /**
+     * Realm.
+     */
+    @ConfigItem(defaultValue = "master")
+    public String realm;
+
+    /**
+     * Keycloak server URL, for example, `https://host:port`.
+     */
+    @ConfigItem
+    public Optional<String> serverUrl;
+
+    /**
+     * Client id.
+     */
+    @ConfigItem
```

Comment:
I propose `admin-cli` by default, we have a default `master` realm configured here too